modules/32_power/IntC/equations.gms

```diff
@@ -119,15 +119,13 @@ q32_limitCapTeChp(t,regi)..
 ***---------------------------------------------------------------------------
 *' Additional grid expansion to integrate VRE are driven linearly by VRE output 
 q32_limitCapTeGrid(t,regi)$( t.val ge 2020 ) .. 
-    vm_cap(t,regi,"gridwind",'1')      !! Technology is now parameterized to yield marginal costs of ~3.5$/MWh VRE electricity
+    vm_cap(t,regi,"gridwindon",'1')    !! Technology is now parameterized to yield marginal costs of ~3.5$/MWh VRE electricity
```

Comment:
(actually, this was only called gridwind for historic reasons. 
It would be clearer if it could be changed to simply one technology "VREgrid". 
but I can understand if you say "why should I clean up this old mess" :-)

Reply:
I actually like cleaning old mess ... (but dont tell anyone else)
Just maybe not right now, so i'll open a separate issue: https://github.com/remindmodel/development_issues/issues/351